sbt/src/sbt-test/site/fopConfig/test

```diff
@@ -1,3 +1,3 @@
 # test PDF output generation with the pdf task and a custom FOP config
 > laika:pdf
-$ exists target/docs/site-fopConfig-0.1.pdf
\ No newline at end of file
+$ exists target/docs/site-fopconfig-0.1.pdf
```

Comment:
Yes, I do (Mac OS X default). However, I was expecting that the plugin actually writes the files in camel-case notation, so that it would work on a case-sensitive fs, too. I just checked and for some reason it does not. So this change is indeed necessary.